content/tutorial/nav.yml

```diff
@@ -1,111 +1,111 @@
-- title: Tutorial
+- title: 學習指南
   items:
     - id: before-we-start-the-tutorial
-      title: Before We Start the Tutorial
+      title: 我們開始這份學習指南之前
       href: /tutorial/tutorial.html#before-we-start-the-tutorial
       forceInternal: true
       subitems:
         - id: what-are-we-building
-          title: What Are We Building?
+          title: 我們要做什麼？
           href: /tutorial/tutorial.html#what-are-we-building
           forceInternal: true
         - id: prerequisites
-          title: Prerequisites
+          title: 先決條件
           href: /tutorial/tutorial.html#prerequisites
           forceInternal: true
     - id: setup-for-the-tutorial
-      title: Setup for the Tutorial
+      title: 教學設定
       href: /tutorial/tutorial.html#setup-for-the-tutorial
       forceInternal: true
       subitems:
         - id: setup-option-1-write-code-in-the-browser
-          title: "Option 1: Write Code in the Browser"
+          title: 設定選項 1：在瀏覽器中寫程式碼
```

Comment:
Do we need quotation marks for option 1 and 2?

Reply:
It's fine for here, not need quotation marks.